components/sidebar-navigation/Component.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from "react";
+import { cnb } from "cnbuilder";
+
+// import specific templates for the component
+import BurgerBar from "../../packages/SVG-library/BurgerBar";
+import Close from "../../packages/SVG-library/Close";
+import LinkItem from "./Components/LinkItem";
+
+/**
+ * Base component
+ *
+ * @param {string} title The component title
+ * ... any other options needed
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export default function sidebarNavigation({
+  asset_assetid,
+  asset_short_name,
+  asset_url,
+  active,
+  menu,
+}) {
+  const [navOpen, setNavOpen] = useState(false);
+  const [showMobileButton, setShowMobileButton] = useState(false);
```

Comment:
Is there a particular reason to add the show/hide mobile button state on top of using TW to hide on LG+? Is it because you prefer to hide it completely in DOM?

Reply:
In working on this I was just trying both things at different times and didn't remove one or the other before PR. I've removed the state in favor of CSS.

Reply:
That makes sense :)